swatch-contracts/src/main/resources/application.properties

```diff
@@ -212,6 +212,8 @@ RBAC_ENDPOINT=${clowder.endpoints.rbac-service.url}
 rhsm-subscriptions.rbac-service.use-stub=${RHSM_RBAC_USE_STUB:false}
 rhsm-subscriptions.rbac-service.stub-permissions=${RHSM_RBAC_STUB_PERMISSIONS:subscriptions:*:*}
 # Note the keystore and truststore values must be prefixed with either "classpath:" or "file:"
+# The .trust-store-* properties are actually synthetic.  They don't appear in the cdappconfig.json file.  Instead,
+# clowder-quarkus-config-source creates references to the Java truststore file we create based off of tlsCaCert file.
```

Comment:
I would not add the same comment in all the application properties since it might confuse more than help. 
The relevant point here is to use the trust-store provided by the clowder config source. What is mentioned in the comment is an implementation detail. 

Moreover, we can find the same information in https://github.com/RedHatInsights/rhsm-subscriptions/pull/3930/commits/bfc30d361244f3e13b1de07637fbf6e41423c085#diff-2e2cbfc13c89a5e8c259890f9ff8977aafcd33398a87740195f9e67e716fba39R59 which is the right place

Reply:
My reasoning for putting that comment in the config files themselves was that `clowder.endpoints...trust-store-path` will never actually appear in the `cdappconfig.json`.  So if someone doesn't know that we generate those properties, when they are integrating a new endpoint they'll find it very confusing.  I wanted to save someone from getting into a "Why isn't my endpoint getting a trust-store-path from Clowder!?".  And on top of that, the Clowder team would have no idea what the developer is talking about since that property is unknown to them.

What do you think?

Reply:
Yes, I see the intention is good to prevent headaches or misunderstandings in the future. However my main concern is that if for some reason this internally changes in the future (it strongly depends to how the Quarkus Clowder config source works plus the cdappconfig.json is structured), we will need to maintain this comment in too many places. 

What about moving this comment into the CONTRIBUTING.md document instead?